packages/esm-patient-chart-app/package.json

```diff
@@ -37,10 +37,12 @@
     "@carbon/icons-react": "^10.18.0",
     "@openmrs/esm-patient-common-lib": "^3.0.0",
     "carbon-components-react": "^7.25.0",
-    "lodash-es": "^4.17.15"
+    "lodash-es": "^4.17.15",
+    "uuid": "^8.3.2"
   },
   "peerDependencies": {
     "@openmrs/esm-framework": "3.x",
+    "@types/uuid": "^8.3.0",
```

Comment:
Move to dev dependencies.

Reply:
Ah, sorry, I didn't pay attention there. Fixed!